ui/packages/catalog-ui-search/src/main/webapp/js/widgets/cesium.polygon.js

```diff
@@ -72,7 +76,7 @@ class PolygonRenderView extends GeometryRenderView {
     this.primitive = new Cesium.PolylineCollection()
     this.cameraMagnitude = this.map.camera.getMagnitude()
     ;(polygons || []).forEach(polygonPoints => {
-      if (!polygonPoints || polygonPoints.length < 3) {
+      if (validateGeo('polygon', polygonPoints).error) {
```

Comment:
❓ does 'polygonPoints' need to be JSON stringified as well?

Reply:
yes :0)